oshdb-util/src/test/java/org/heigit/bigspatialdata/oshdb/util/geometry/OSHDBGeometryBuilderTest.java

```diff
@@ -160,6 +162,28 @@ public void testRelationGetGeometryIncomplete() {
     assertEquals("GeometryCollection", result.getGeometryType());
     assertEquals(1, result.getNumGeometries());
   }
+  
+  @Test
+  public void testBoundingBoxGetGeometry() {
+    OSHDBBoundingBox bbx = new OSHDBBoundingBox(0, 0, 0, 0);
```

Comment:
maybe better start the test with the "regular" case, and then test the more specific corner cases like point/line degenerated bbox? also a comment line would be nice here to explain why these need to be tested differently.

Reply:
code-quality is really high on the agenda these days. I like it :+1: !!! Many things to learn for me.